README.md

```diff
@@ -1,35 +1,65 @@
-# Nova++
+# HERACLES++
 
 ## How to get sources
 
-To get the full source of nova and external libraries:
+To get the full source of HERACLES++ and external libraries:
 
 ```bash
-git clone --recurse-submodules https://gitlab.maisondelasimulation.fr/lrousselhard/nova.git
+git clone --recurse-submodules https://github.com/Maison-de-la-Simulation/heraclespp.git
 ```
 
-In case the repository has already been cloned by
+In case the repository has already been cloned by:
 
 ```bash
 git clone https://gitlab.maisondelasimulation.fr/lrousselhard/nova.git
 ```
 
-You can retrieve the dependencies (PDI, Kokkos,...) with
+You can retrieve the dependencies (PDI, Kokkos,...) with:
 
 ```bash
 git submodule init && git submodule update
 ```
 
 ## How to build
 
-A straightforward way to build Nova++ is to assume that all dependencies are available in the environment. In this case, one can compile the advection setup in a directory called `build` with the two following commands
```

Comment:
Do you mind if we also keep this version ? I have in mind to have two sections:
- How to build with pre-installed dependencies (recommended)
- How to build with vendored dependencies